04_numeronym/n0npax/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func shorten(word string) string {
```

Comment:
Don't feel the need to change this, since this is already fine as is.
Just wanted to point out you can also assign `wordlen` in the `if` block...
```
if wordlen := len(word); wordlen < 4 {
  return word
} else {
  return fmt.Sprintf("%c%d%c", word[0], wordlen-2, word[wordLen-1])
}
```

Reply:
thanks for hint